packages/edit-widgets/src/style.scss

```diff
@@ -40,3 +40,31 @@ body.gutenberg_page_gutenberg-widgets {
 		height: 100%;
 	}
 }
+
+/**
+ * Animations
+ */
+
+// These keyframes should not be part of the _animations.scss mixins file.
+// Because keyframe animations can't be defined as mixins properly, they are duplicated.
+// Since hey are intended only for the editor, we add them here instead.
+@keyframes edit-post__loading-fade-animation {
```

Comment:
This makes me think this animations should probably be moved to the movers stylesheet, if it's common to all usage of the editor movers?

Reply:
It looks like this animation is reused in a few places, for example for the transient state in the file block, and the saving state for the save indicator. I imagine we'll want to use this in additional places, so it's nice with a mixin to help us create consistent animations.

but it does feel weird to keep duplicating these keyframes. 

As a fix to the problem, 👍 👍 from me, but CC: @kjellr if you have any ideas as to how we can better reuse this animation and avoid the duplication.

Reply:
Is there a way to move this animation to the `Animate` component?

Reply:
Yeah, I was thinking it'd be great to consolidate all of the keyframe animations that we reuse into a single place — they're currently all over the place. The animate component might be a good place for that. 

That definitely seems like something that could be handled in a separate PR though. This one seems 👍 to me.